pkg/adaptor/proxy/service.go

```diff
@@ -44,13 +44,18 @@ func newClient(conn net.Conn) *client {
 }
 
 type proxyService struct {
-	agentClient   *client
-	maxRetries    int
-	retryInterval time.Duration
+	agentClient    *client
+	maxRetries     int
+	retryInterval  time.Duration
+	criSocketPath  string
+	criClient      *containerd.Client
+	criImageClient runtime.ImageServiceClient
```

Comment:
We don't need to have two separated fields for criClient and criImageClient. We only need to have ImageServiceClient as criClient.

If we want to use RuntimeServiceClient in the future, we can define a new struct like this
```
type criClient struct {
	criapi.RuntimeServiceClient
	criapi.ImageServiceClient
}
```

Reply:
Yes, struct is more convenient for future extension.

Reply:
Fixed